### ISSUE TITLE
Serious error in Standoc due to Algolia error

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-standoc#91]
When running the latest gems on an OGC document I was presented with this error:

```sh
$ bundle exec metanorma -t ogc -x html,doc pipelineml/document.adoc 
asciidoctor: ERROR: pipelineml/sections/clause_5_conventions.adoc: line 7: ISO 19136:2007 sections do not support nested sections
asciidoctor: ERROR: pipelineml/sections/clause_5_conventions.adoc: line 99: ISO 19136:2007 sections do not support nested sections
asciidoctor: ERROR: pipelineml/sections/clause_5_conventions.adoc: line 114: ISO 19136:2007 sections do not support nested sections
asciidoctor: ERROR: pipelineml/sections/clause_5_conventions.adoc: line 166: ISO 19136:2007 sections do not support nested sections
asciidoctor: ERROR: pipelineml/sections/clause_5_conventions.adoc: line 190: ISO 19136:2007 sections do not support nested sections
asciidoctor: WARNING: pipelineml/sections/annex-a.adoc: line 11: invalid style for example block: requirement
asciidoctor: WARNING: pipelineml/sections/annex-a.adoc: line 14: invalid style for example block: verification
[relaton] Info: detecting backends:
[relaton] processor "relaton_iec" registered
[relaton] processor "relaton_iso" registered
[relaton] processor "relaton_ietf" registered
[relaton] processor "relaton_gb" registered
[relaton] processor "relaton_nist" registered
[relaton] processor "relaton_itu" registered
fetching ISO: ISO 19101-1...
WARNING: no match found online for ISO: ISO 19101-1:2014. The code must be exactly like it is on the standards website.
The provided document part may not exist, or the document may no longer be published in parts.
fetching ISO: ISO 19103...
WARNING: no match found online for ISO: ISO 19103:2005. The code must be exactly like it is on the standards website.
If you wanted to cite all document parts for the reference, use "ISO: ISO 19103 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
fetching ISO: ISO 19107...
WARNING: no match found online for ISO: ISO 19107:2003. The code must be exactly like it is on the standards website.
If you wanted to cite all document parts for the reference, use "ISO: ISO 19107 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
fetching ISO: ISO 19109...
WARNING: no match found online for ISO: ISO 19109:2015. The code must be exactly like it is on the standards website.
If you wanted to cite all document parts for the reference, use "ISO: ISO 19109 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
OGC: OGC 18-005r4 does not have a recognised prefix: IEC, ISO, IETF, CN, NIST, ITU
OGC: OGC 18-005r4 does not have a recognised prefix: IEC, ISO, IETF, CN, NIST, ITU
fetching ISO: ISO 19115...
WARNING: no match found online for ISO: ISO 19115:2014. The code must be exactly like it is on the standards website.
If you wanted to cite all document parts for the reference, use "ISO: ISO 19115 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
OGC: OGC 07-036r1 does not have a recognised prefix: IEC, ISO, IETF, CN, NIST, ITU
OGC: OGC 07-036r1 does not have a recognised prefix: IEC, ISO, IETF, CN, NIST, ITU
bundler: failed to load command: metanorma (/Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/bin/metanorma)
NameError: uninitialized constant Asciidoctor::Standoc::Lists::Algolia
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.2.2/lib/asciidoctor/standoc/ref.rb:112:in `rescue in fetch_ref'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.2.2/lib/asciidoctor/standoc/ref.rb:106:in `fetch_ref'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.2.2/lib/asciidoctor/standoc/ref.rb:136:in `refitem'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.2.2/lib/asciidoctor/standoc/ref.rb:189:in `reference1'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.2.2/lib/asciidoctor/standoc/ref.rb:200:in `block (2 levels) in reference'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.2.2/lib/asciidoctor/standoc/ref.rb:199:in `each'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/metanorma-standoc-1.2.2/lib/asciidoctor/standoc/ref.rb:199:in `block in reference'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.3/lib/nokogiri/xml/builder.rb:293:in `initialize'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.3/lib/nokogiri/xml/builder.rb:254:in `new'
  /Users/me/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/nokogiri-1.10.3/lib/nokogiri/xml/builder.rb:254:in `with'
```

This seems relevant to Relaton. Relaton should NEVER be throwing Algolia errors!!

This will need a re-release.

**Comment by ronaldtse** [comment]
@opoudjis I can't seem to use metanorma-standoc 1.2.3 which fixes this error. My Gemfile only lists `metanorma-cli`.

```
Using metanorma-standoc 1.2.2
Using metanorma-acme 1.2.3
Using metanorma-csand 1.2.2
Using metanorma-csd 1.2.2
Using metanorma-ogc 0.1.3
Using metanorma-unece 0.1.3
Using metanorma-iso 1.2.3
Using metanorma-m3d 1.2.2
Using metanorma-gb 1.2.3
Using metanorma-ietf 1.0.5
Using metanorma-itu 0.1.3
Using metanorma-nist 0.1.3
Using metanorma-cli 1.2.3
```

Seems that there is some gem that hasn't been released with 1.2.3?

**Comment by opoudjis** [comment]
Yes, Metanorma-NIST. Has just been fixed.

@CAMOBAP795, I *really* need help with release management. The issue here has been a major release of Relaton, which impacts Metanorma-NIST as well as Metanorma-Standoc, so I need to release both whenever there is a major release of Relaton.
